### PR TITLE
Update k8s manifests with target flags

### PR DESCRIPTION
**Pull request by weeco**
After https://github.com/cortexproject/cortex/pull/1262 has been merged, one has to set target flags for each microservice. I updated the k8s manifests to do so.

While doing so I noticed an inconsistency of the target flags. `alertmanager` had been used while beforehand `table-manager` and `query-frontend` were used. Therefore I also updated the alertmanager module name to achieve consistent target flags - before it's too late.

### CODE REVIEW

File: pkg/cortex/modules.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (m moduleName) String() string {
 	case Configs:
 		return "configs"
 	case AlertManager:
-		return "alertmanager"
+		return "alert-manager"
```
Comment (jtlisi) [inline pkg/cortex/modules.go]: Why are you hyphenating here?

Comment (weeco) [reply pkg/cortex/modules.go]: Please see my comment in the PR submit

Comment (weeco) [reply pkg/cortex/modules.go]: Actually I believe you are right, Alertmanager is just one word. I was confused due to the variable name with an uppercased "Manager".

**Comment by weeco** [comment]
Done

**Pull request merged**